internal/backup.go

```diff
@@ -67,53 +66,51 @@ func (backup *Backup) SentinelExists() (bool, error) {
 
 // TODO : unit tests
 func (backup *Backup) FetchSentinel(sentinelDto interface{}) error {
-	sentinelDtoData, err := backup.fetchStorageBytes(backup.getStopSentinelPath())
+	sentinelDtoData, err := backup.fetchStorageStream(backup.getStopSentinelPath())
 	if err != nil {
 		return errors.Wrap(err, "failed to fetch sentinel")
 	}
-	err = json.Unmarshal(sentinelDtoData, sentinelDto)
+	err = einJSON.Unmarshal(sentinelDtoData, sentinelDto)
```

Comment:
Maybe add the flag/config setting to control this behaviour since it is quite new?